src/Block/FileBlock.php

```diff
@@ -74,6 +75,10 @@ protected function provideBlockSchema()
      */
     public function getSummaryThumbnail()
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('1.13.0', 'Will be removed without equivalent functionality to replace it.');
+        });
```

Comment:
```suggestion
        Deprecation::notice('3.1.0', 'Will be removed without equivalent functionality to replace it.');
```
Fix version number.
Also don't wrap it in `withNoReplacement()` - that's only used for something we're calling from internal code, which isn't the case here. Wrapping with this method would actually _suppress_ the deprecation warning, which we don't want to do for this method.

Reply:
Updated